src/python/bot/tasks/update_task.py

```diff
@@ -169,6 +169,8 @@ def run_platform_init_scripts():
   plt = environment.platform()
   if plt == 'ANDROID':
     android_init.run()
+  elif plt == 'ANDROID_KERNEL':
+    linux_init.run()
```

Comment:
this should be android_init.run as we initialize device.

Reply:
Ok, I did that because we use TF which flashes the device already. But I will change as per your comment to support the general case.